src/coreComponents/mesh/generators/InternalMeshGenerator.cpp

```diff
@@ -140,7 +140,7 @@ void InternalMeshGenerator::postProcessInput()
     }
     if( failFlag )
     {
-      GEOS_ERROR( "vertex/element mismatch InternalMeshGenerator::ReadXMLPost()" );
+      GEOS_ERROR( getDataContext() << ": vertex/element mismatch." << generalMeshErrorAdvice );
```

Comment:
```suggestion
      GEOS_ERROR( getDataContext() << ": vertex/element mismatch.\n" << generalMeshErrorAdvice );
```
Then this would be it?